### PR TITLE
split scheduler priorities into separate files

### PR DESCRIPTION
**Pull request by gmarek**
In the current state it's really hard to find a thing one is looking for, if he doesn't know already know where to look. cc @davidopp

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35612)

### CODE REVIEW

**Comment by gmarek** [comment]
@wojtek-t PTAL

File: plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+// This is a reasonable size range of all container images. 90%ile of images on dockerhub drops into this range.
+const (
+	mb         int64 = 1024 * 1024
+	minImgSize int64 = 23 * mb
+	maxImgSize int64 = 1000 * mb
+)
+
+// Also used in most/least_requested nad metadata.
+// TODO: despaghettify it
+func getNonZeroRequests(pod *api.Pod) *schedulercache.Resource {
```
Comment (wojtek-t) [inline plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go]: Can you move it to priority/util/non_zero.go? Or will it introduce a cycle?

**Comment by wojtek-t** [comment]
lgtm

**Comment by gmarek** [comment]
Applying label per https://github.com/kubernetes/kubernetes/pull/35612#issuecomment-256919208

**Pull request merged**